### DISCUSSION TITLE
Windows App SDK 1.4.2 (1.4.231008000)

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#3906] [Announcements]
## Windows App SDK 1.4.2 (1.4.231008000)

Windows App SDK 1.4.2 is a stable servicing release that fixes critical bugs for 1.4. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.4 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/stable-channel#version-142-14231008000) for the list of issues fixed in this release.

### Updating to this version

The 1.4.2 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.4.231008000

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.4.2'>Windows App SDK 1.4.2 (1.4.231008000)</a>.</em>

**Comment by ghost1372** [comment]
Ok when symbols will be available? Also source codes...

**Reply by duncanmacmichael** [reply]
Coming soon! I double checked for you and it seems like the final steps to get the source and symbols published are taking a bit longer than expected. Still coming, just going through final hoops internally. Thanks for your patience!
